pkg/reconciler/serverlessservice/serverlessservice.go

```diff
@@ -285,6 +285,9 @@ func (r *reconciler) reconcilePublicEndpoints(ctx context.Context, sks *netv1alp
 }
 
 func (r *reconciler) privateService(sks *netv1alpha1.ServerlessService) (*corev1.Service, error) {
+	// The code below is for backwards compatibility, when we had
+	// GenerateName for the private services.
+	// TODO(vagababov) remove this in 0.11.
```

Comment:
We don't have anything in here to move things to the new name, so we can't just remove this.

Reply:
Oh we can, it's just additional churn  in the service creation that will take place on the old services.
As in I can remove this code now and nothing will fail, but we'll create one more service. Considering support of two versions down, at the point of 0.11 we can delete this since _more_ revisions will change by then.

Reply:
I'm fine with leaving this lookup via selector 🤷‍♂

Reply:
🤷‍♀ 
removed